packages/entities/src/lib/many.query.ts

```diff
@@ -95,3 +98,36 @@ export function selectMany<S extends EntitiesState<Ref>, Ref>(
     distinctUntilArrayItemChanged()
   );
 }
+
+export function selectManyByPredicate<
+  S extends EntitiesState<Ref>,
+  R extends getEntityType<S, Ref>[],
+  K extends keyof getEntityType<S, Ref>,
+  Ref extends EntitiesRef = DefaultEntitiesRef
+>(
+  predicate: ItemPredicate<getEntityType<S, Ref>>,
+  options?: {
+    pluck?: K | ((entity: getEntityType<S, Ref>) => R);
+  } & BaseEntityOptions<Ref>
+): OperatorFunction<S, getEntityType<S, Ref>[]> {
+  const { ref: { entitiesKey, idsKey } = defaultEntitiesRef, pluck } =
+    options || {};
+
+  return pipe(
```

Comment:
Now I'm wondering why not just delegate the work to selectMany(ids)

Reply:
true